src/components/contact-form/contact-form.js

```diff
@@ -0,0 +1,58 @@
+import Component from '../../app/js/base/Component';
+
+class ContactForm extends Component {
+    form
+    sendButton
+    errorField
+    loader
+
+    constructor(element) {
+        super(element);
+
+
+        this.form = document.getElementById('contact-form');
+        this.form.addEventListener('submit', this.handleSubmitForm);
+        this.sendButton = this.getElement('send-button');
+
+        this.errorField = document.querySelector('.contact-form__error-field');
```

Comment:
Желательно делать поиск внутренних элементов только внутри компонента. В случаях, когда на странице может быть несколько одинаковых компонентов, такой поиск будет пересекаться с другими компонентами и это может вызывать баги
```suggestion
        this.form = this.getElement('controls');
        this.form.addEventListener('submit', this.handleSubmitForm);
        this.sendButton = this.getElement('send-button');

        this.errorField = this.getElement('error-field');
```

Reply:
Понял, учту в дальнейшем. Спасибо!)